src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -123,6 +123,64 @@ function Test-AnalysisServicesServer
 	}
 }
 
+<#
+.SYNOPSIS
+Tests scale up and down of Analysis Services server (B1 -> S2 -> S1).
+#>
+function Test-AnalysisServicesServerScaleUpDown
+{
+    param
+	(
+		$location = "West US"
```

Comment:
I updated the change log.
Sent you an email with questions about location. It is not really clear what and how should be implemented.